tracepoint/src/native.rs

```diff
@@ -293,9 +276,9 @@ impl Drop for UserEventsDataFile {
     }
 }
 
-/// Represents a tracepoint registration.
+/// Low-level API: Represents a tracepoint registration.
 pub struct TracepointState {
```

Comment:
Note: Moved from eventheader crate. The `write_eventheader` method was left behind. Doc comments were reviewed and extended as appropriate since this is now public (was "internal").